projects/core/src/cms/page/content-page-meta.resolver.ts

```diff
@@ -47,8 +65,6 @@ export class ContentPageMetaResolver
    * The home page label is resolved from the translation service.
    */
   resolveBreadcrumbs(): Observable<BreadcrumbMeta[]> {
-    return this.translation
-      .translate('common.home')
-      .pipe(map((label) => [{ label: label, link: '/' }] as BreadcrumbMeta[]));
+    return this._breadcrumbs$;
```

Comment:
what about changing to: 

```typescript
/**
   * Resolves a single breacrumb item to the home page for each `ContentPage`.
   * The home page label is resolved from the translation service.
   */
  resolveBreadcrumbs(): Observable<BreadcrumbMeta[]> {
    return this.breadcrumbs$;
  }

  private get homeBreadcrumb$(): Observable<BreadcrumbMeta[]> {
    return this.translation
      .translate('common.home')
      .pipe(map((label) => [{ label: label, link: '/' }] as BreadcrumbMeta[]));
  }

  private get breadcrumbs$(): Observable<BreadcrumbMeta[]> {
    return combineLatest([
      this.homeBreadcrumb$,
      defer(() => this.routingPageMetaResolver.resolveBreadcrumbs()),
    ]).pipe(
      map(
        (breadcrumbs) => breadcrumbs.flat(),
        shareReplay({ bufferSize: 1, refCount: true })
      )
    );
  }
```

I find it easier to read, and like the fact that some details are below in the file. Not sure if it gives the same performance, as we like to reuse the observable.s

Reply:
I also like to read the logic in the file top-to-bottom (general-to-detail).
I can add ts-ignore comment to allow for putting properties after methods. But anyway `homeBreadcrumb$` _must_ bo before `breadcrumbs$`.

Getters return fresh observable on each subscription (drops whole benefit of shareReplay). I'd like to avoid it.